new_model_seen_template/README.md

```diff
@@ -0,0 +1,48 @@
+| dataset_name| str: Name for analysis directory, which contains the notebooks being run. For example, this analysis directory is named "new_model_seen_template".|
+| raw_compendium_filename | str: Downloaded compendium gene expression data file. The input dataset should be a matrix that is sample x gene. The file should tab-delimited. The gene ids should be HGNC symbols (if using human data) or PA numbers (if using *P. aeruginosa* data). The input dataset should be generated using the same platform as the model you plan to use (i.e. RNA-seq or array). The expression values are expected to have been uniformly processed and can be estimated counts (RNA-seq) or log2 expression (array).|
+| experiment_to_sample_filename | str:  File mapping experiment ids to sample ids|
+| metadata_delimiter | str:  Delimiter ("," or "\t") used in the metadata file that maps experiment id to sample ids|
+| experiment_id_colname | str:  Header of experiment-to-sample metadata file corresponding to the column containing experiment ids. This is used to extract gene expression data associated with project_id|
+| sample_id_colname | str:   Header of experiment-to-sample metadata file to indicate column containing sample ids.This is used to extract gene expression data associated with project_id|
+| project_id | str:  The experiment id to use as template experiment. This experiment must be contained within the training dataset that was used to train the VAE. The id is using the values found in the <metadata_experiment_colname> column of the <experiment_to_sample_filename>. This <project_id> corresponds to a group of samples that were used to test an single hypothesis. This parameter is used to pull out the template expression data from the normalized compendium and to simulate data based on this experiment.|
+| num_simulated| int: The number of experiments to simulate. Experiments are simulated by shifting the template experiment in the latent space.|
+| DE_method| str: "limma" or "DESeq". Differential expression method to use.|
+| NN_architecture | str: Name of neural network architecture to use. Format 'NN_<intermediate layer>_<latent layer>'|
+| learning_rate| float: Step size used for gradient descent. In other words, it's how quickly the  methods is learning|
+| batch_size | str: Training is performed in batches. So this determines the number of samples to consider at a given time|
+| epochs | int: Number of times to train over the entire input dataset|
+| kappa | float: How fast to linearly ramp up KL loss|
+| intermediate_dim| int: Size of the hidden layer|
+| latent_dim | int: Size of the bottleneck layer|
+| epsilon_std | float: Standard deviation of Normal distribution to sample latent space|
+| validation_frac | float: Fraction of samples to use for validation in VAE training|
+
+Parameters for intermediate files created. Names of files need to be specified:
+*Note: These file paths need to be created for the user in order to be used.*
```

Comment:
What does this mean?

Reply:
This was meant to say that the directory that you want to use to store intermediate data files need to be created. Otherwise you'll get an error about the location not existing.